CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [10.22.2]
+### Changed
+- Restore changes from v10.22.0
+- Update `SmartTransactionsController` to v3 ([#16511](https://github.com/MetaMask/metamask-extension/pull/16511))
+  - Drops support for Node.js v12
+  - Fixes deadlock encountered while performing Swaps with STX on unsupported testnets
```

Comment:
Nit: This changelog is for end-users, who do not understand what controllers are. We should try to write these changes in a way that is understandable for people outside of our team.

I've updated this to avoid references to internals, and changed this to be imperative for consistency

```suggestion
- Fix deadlock encountered while performing Swaps on unsupported testnets ([#16511](https://github.com/MetaMask/metamask-extension/pull/16511))
```